### ISSUE TITLE
embree patch failed

### ISSUE THREAD

**Issue by microcoder** [bartoszek/AUR-blender-git#7]
I cannot build it.

```
==> Starting prepare()...
Checking patch build_files/cmake/Modules/FindEmbree.cmake...
Hunk #2 succeeded at 79 (offset 9 lines).
Checking patch intern/cycles/blender/CMakeLists.txt...
error: while searching for:
  )
endif()

set(ADDON_FILES
  addon/__init__.py
  addon/engine.py

error: patch failed: intern/cycles/blender/CMakeLists.txt:67
error: intern/cycles/blender/CMakeLists.txt: patch does not apply
==> ERROR: A failure occurred in prepare().
```

if remove patch `embree.patch` from PKGBUILD then build end without mistake:

```
prepare() {
  cd "$srcdir/blender"
  # update the submodules
  git submodule update --init --recursive --remote
  # git apply -v "${srcdir}"/{embree,usd_python,openexr3}.patch
  git apply -v "${srcdir}"/{usd_python,openexr3}.patch
}
```

**Comment by bartoszek** [comment]
Fixed in  59879c923e1a6393763c34ce6042914fa36985c7

Skipping `embree.patch` will build an internal static version of Embree provided in the blender source tree.

**Comment by microcoder** [comment]
> Fixed in [59879c9](https://github.com/bartoszek/AUR-blender-git/commit/59879c923e1a6393763c34ce6042914fa36985c7)

Thanks!
